BraveWallet/Crypto/Portfolio/EditUserAssetsView.swift

```diff
@@ -73,6 +86,35 @@ struct EditUserAssetsView: View {
         ) {
           ForEach(tokenStores, id: \.token.id) { store in
             EditTokenView(assetStore: store)
+              .osAvailabilityModifiers { content in
+                if !store.isCustomToken {
+                  content
+                } else {
```

Comment:
We shouldn't include non-`#available` checks within this modifier to avoid lifetime side effects